app/src/main/java/com/orange/ods/app/ui/components/cards/CardVerticalImageFirst.kt

```diff
@@ -46,26 +48,27 @@ fun CardVerticalImageFirst(customizationState: CardCustomizationState) {
                 .padding(dimensionResource(id = com.orange.ods.R.dimen.spacing_m))
```

Comment:
Not from this PR but we have a problem of padding on this screen so if we can replace this line by:
```kotlin
 .padding(horizontal = dimensionResource(id = com.orange.ods.R.dimen.spacing_m))
```

Add this modifier to the `OdsVerticalImageFirstCard`:
```kotlin
modifier = Modifier.padding(top = dimensionResource(id = com.orange.ods.R.dimen.spacing_m))
```

 Remove the `Spacer` before `CodeImplementationColumn` and replace it by the following modifier:
```kotlin
  modifier = Modifier.padding(
      top = dimensionResource(id = com.orange.ods.R.dimen.spacing_s),
      bottom = dimensionResource(id = com.orange.ods.R.dimen.spacing_m)
  )
```

Same problem in `CardVerticalHeaderFirst.kt`, `CardHorizontal` and `CardSmall`